financial-connections/src/main/java/com/stripe/android/financialconnections/features/manualentry/ManualEntryViewModel.kt

```diff
@@ -143,7 +143,6 @@ internal class ManualEntryViewModel @AssistedInject constructor(
             pollAttachPaymentAccount(
                 sync = sync,
                 activeInstitution = null,
-                consumerSessionClientSecret = null,
```

Comment:
This is the key change in case it gets obscured with the refactor -> manual entry was not passing the cs_secret.